config/routes.rb

```diff
@@ -348,19 +348,24 @@
       get '/in_person/:step' => 'in_person#show', as: :in_person_step
       put '/in_person/:step' => 'in_person#update'
 
-      get '/inherited_proofing' => 'inherited_proofing#index'
-      get '/inherited_proofing/:step' => 'inherited_proofing#show', as: :inherited_proofing_step
-      put '/inherited_proofing/:step' => 'inherited_proofing#update'
-      get '/inherited_proofing/return_to_sp' => 'inherited_proofing#return_to_sp'
-      get '/inherited_proofing/cancel/' => 'inherited_proofing_cancellations#new', as: :inherited_proofing_cancel
-      put '/inherited_proofing/cancel' => 'inherited_proofing_cancellations#update'
-      delete '/inherited_proofing/cancel' => 'inherited_proofing_cancellations#destroy'
-
       # deprecated routes
       get '/confirmations' => 'personal_key#show'
       post '/confirmations' => 'personal_key#update'
     end
 
+    # Inherited Proofing (IP)-specific routes.
+    scope '/verify/inherited_proofing', module: 'idv', as: 'idv_inherited_proofing' do
```

Comment:
Would it have been possible to keep this inside the `/verify` scope, and then define the nested scope as ...

```rb
scope '/inherited_proofing', module: 'idv', as: 'inherited_proofing' do
```

Reply:
Humm. That was a toss up for me because the /verify scope just seems _so_ cluttered, i decided to segregate it :(.